### PR TITLE
Update nginx configuration.

### PR DESCRIPTION
**Pull request by igorpeshansky**

### CODE REVIEW

File: etc/nginx/conf.d/status.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+server {
+    listen 80;
+    server_name local-stackdriver-agent.stackdriver.com;
+    location /nginx_status {
+        stub_status on;
+        access_log   off;
```
Comment (coreykosak) [inline etc/nginx/conf.d/status.conf]: weird extra space

File: etc/collectd.d/nginx.conf [hunk 2]
```diff
@@ -5,5 +5,5 @@ LoadPlugin nginx
 <Plugin "nginx">
     # When using non-standard Nginx configurations, replace the below with
     #URL "http://NGINX_HOST:NGINX_PORT/nginx_status"
-    URL "http://localhost/nginx_status"
+    URL "http://local-stackdriver-agent.stackdriver.com:80/nginx_status"
```
Comment (coreykosak) [inline etc/collectd.d/nginx.conf]: Is this safe to use? I notice the address resolves correctly on my tests on EC2 and GCE, but not on my local (internal) Google workstation. Maybe that's just due to some paranoid Google setting?

**Comment by coreykosak** [comment]
Otherwise, LGTM

File: etc/collectd.d/nginx.conf (hunk 2 above)
Comment (igorpeshansky) [inline etc/collectd.d/nginx.conf]: Running `dig local-stackdriver-agent.stackdriver.com` contains the following:
```
;; ADDITIONAL SECTION:
.			0	IN	TXT	"Error" "Answer blocked by policy."
```
So I'd say we're good.

**Comment by igorpeshansky** [comment]
Fixed the spacing, too. Thanks.

**Pull request merged**